### ISSUE TITLE
Google Play link broken 

### ISSUE THREAD

**Issue by sanicki** [TobiasBielefeld/Simple-Solitaire#172]
Is the app still available on Google Play? Your link appears to no longer be working.

I don't mind using F-Droid, but I'd like to recommend Simple Solitaire to some less tech-fluent friends for whom that would be a hurdle.

**Comment by vphantom** [comment]
From what I can tell, it's been removed from Google Play entirely. 😕

**Comment by AnEnglishmanInNorway** [comment]
> From what I can tell, it's been removed from Google Play entirely. 😕

Yes, it looks that way to me as well, which is really really sad. This is IMHO the best Solitaire collection out there, with no adverts and no unnecessary permissions. Would you please consider putting it back again, @TobiasBielefeld? I've made a fork with a view to doing it myself, but it seems a lot of effort (for a C#/C++ programmer using Visual Studio!) to do what has already been done once
